### PR TITLE
ENH: Separate HASI Python package YML for CI testing

### LINKED ISSUE #43: CI testing for the `hasi` package
Add a new GitHub Action workflow to test the package in *src/hasi*. Testing

- Building the package
- Running pytest

Reference:https://docs.github.com/en/actions/guides/building-and-testing-python

### PR DESCRIPTION
**Pull request by tbirdso**
Closes #43 .

### CODE REVIEW

File: .github/workflows/test-python-hasi.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Build, test, package
```
Comment (dzenanz) [inline .github/workflows/test-python-hasi.yml:1]: I don't see build and package steps. So maybe `Test Python HASI`?

File: .github/workflows/test-python-hasi.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        os: [ubuntu-18.04, windows-2019, macos-10.15]
+        include:
+          - os: ubuntu-18.04
+          - os: windows-2019
+          - os: macos-10.15
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Move to hasi module directory
```
Comment (dzenanz) [inline .github/workflows/test-python-hasi.yml:23]: Could this command be merged with the last one (`pytest .`)? If not, maybe rename to `change directory to HASI source`?

**Review by dzenanz — COMMENTED** [review]
Mostly looks good

File: .github/workflows/test-python-hasi.yml (hunk 2 above)
Comment (tbirdso) [reply .github/workflows/test-python-hasi.yml:23]: Good point. Added `cd` command to final `pytest` step.

**Review by dzenanz — APPROVED** [review]

**Comment by tbirdso** [comment]
Working to investigate the source of the unresolved `pytest` issue. Seeing an attempt to access an out-of-bounds point ID in `itk::KdTree` nearest-neighbor resampling:
> template_mesh.SetPoint(i, target_mesh.GetPoint(neighbors.GetElement(0)))
>E           RuntimeError: /work/ITK-source/ITK/Modules/Core/Common/include/itkPointSet.hxx:204:
>E           ITK ERROR: Mesh(0x5558f2c38640): Point id doesn't exist: 18446744073709551615

This issue seems to only be present in the Linux build and in this PR. Tests are passing in the `master `branch`:
> src/hasi/test_meshtomeshregistration.py ....                             [100%]
================== 4 passed, 1 warning in 1115.50s (0:18:35) ===================

**Comment by tbirdso** [comment]
Going to go ahead and merge this branch. Changes made here only update CI while failures appear to result from underlying issues with KdTree.

**Pull request merged**